aziotctl/src/main.rs

```diff
@@ -48,4 +50,6 @@ enum Options {
 
     /// List the checks that are run for 'aziotctl check'
     CheckList(check_list::CheckListOptions),
+    ///Use system helper commands
```

Comment:
Missing empty line between the enum variants.

Reply:
and missing a space between `///` and `Use`